README.md

```diff
@@ -1,6 +1,6 @@
 # Project Evergreen
 
-Project Evergreen is a group project for the class CSE312: Web Applications at the University at Buffalo in the Fall 2024 semester. This project is intended to become the web application for the UB IEEE student chapter automated greenhouse chapter. 
+Project Evergreen is a group project for the class CSE312: Web Applications at the University at Buffalo in the Fall 2024 semester. This project is intended to become the web application for the UB IEEE student chapter automated greenhouse chapter.
```

Comment:
Removing the trailing spaces isn't bad, but I'd un-do it anyway just to avoid the conflict with Eric's rewriting of the README on merge

Reply:
done